easybuild/easyblocks/s/superlu.py

```diff
@@ -93,6 +93,13 @@ def configure_step(self):
             openblas_lib = os.path.join(get_software_root('OpenBLAS'), get_software_libdir('OpenBLAS'), "libopenblas.a")
             self.cfg.update('configopts', '-DBLAS_LIBRARIES="%s;-pthread"' % openblas_lib)
 
+        elif toolchain_blas == 'FlexiBLAS':
+            # Unfortunately, FlexiBLAS is not recognized by FindBLAS from CMake,
+            # we have to specify the FlexiBLAS library manually
+            openblas_lib = os.path.join(get_software_root('FlexiBLAS'), get_software_libdir('FlexiBLAS'), 
+                "libflexiblas.so")
+            self.cfg.update('configopts', '-DBLAS_LIBRARIES="%s;-pthread"' % openblas_lib)
```

Comment:
This doesn't look right, `openblas_lib` has no leading `-` so why does `pthread` need one?